packages/ember-data/tests/unit/transform/number_test.js

```diff
@@ -20,7 +20,7 @@ test("#deserialize", function() {
   equal(transform.deserialize(undefined),       null);
   equal(transform.deserialize("1.1"),           1.1);
   equal(transform.deserialize(1.1),             1.1);
-  equal(transform.deserialize(new Number(1.1)), 1.1);
+  equal(transform.deserialize(Number(1.1)), 1.1);
```

Comment:
It's a good global rule, but I think that here we are specifically testing for people who passed in a number object.